app/controllers/clients_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @client_projects = @client.projects
```

Comment:
if you do not need to use it multiple places then you can just do @client.projects in views.